### PR TITLE
Allow null value for $userId

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | -

### CODE REVIEW

File: src/AccessCheckerInterface.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ interface AccessCheckerInterface
     /**
      * Checks if the user with the ID given has the specified permission.
      *
-     * @param int|string $userId the user ID representing the unique identifier of a user.
+     * @param int|string|null $userId the user ID representing the unique identifier of a user. If ID is null, it means user is a guest.
```
Comment (vjik) [inline src/AccessCheckerInterface.php]:
```suggestion
     * @param int|string|null $userId The user ID representing the unique identifier of a user.
     * If ID is null, it means user is a guest.
```

**Review by vjik — CHANGES_REQUESTED** [review]
Need changelog with @roxblnfk (https://github.com/yiisoft/access/pull/9)

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Looks 👍 I think we can release it as a patch version. Would you please add a line for CHANGELOG?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Yii Access
 
+## 1.0.1 under development
+
+- Chg #13: Allowed passing a null value in $userId to check the permission of the guest user (@rustamwin)
```
Comment (vjik) [inline CHANGELOG.md]:
```suggestion
- Chg #9: Type of `$userId` in the method `AccessCheckerInterface::userHasPermission()` is specified (@roxblnfk)
- Chg #13: Allowed passing a null value in `$userId` to check the permission of the guest user (@rustamwin)
```

**Review by vjik — APPROVED** [review]

**Pull request merged**